### DISCUSSION TITLE
Question about translation.

### DISCUSSION THREAD

**Discussion by ericxuo** [eteran/nedit-ng#55] [General]
Hi,

For example, when the "Find" or "Search And Replace" pop-up windows are open, some fields are not correctly aligned in the french version (see bottom of both pictures).
To get a better "look and feel" result, I could add spaces at the end of the "Case Sensitive" field.
Do you agree this is the correct way to get a better result ?

Find window : 
![search](https://user-images.githubusercontent.com/9089499/44620875-5210d700-a89c-11e8-9f6e-61fb8ae91258.png)

Search And Repalce window : 
![searchandreplace](https://user-images.githubusercontent.com/9089499/44620876-5210d700-a89c-11e8-9929-459cedcf0cd1.png)

I'd prefer to reduce the length of "(Utilisez la flèche vers le haut pour rappeler la chaîne précédente)" but french is verbose ...

Regards.

Xuo.

**Comment by eteran** [comment]
Good question. I'll think about it. But I've also been considering a slightly more vertical layout, which in itself would solve the issue in a different way.

I'll let you know what I come up with :-)
